src/components/Explore/utils.ts

```diff
@@ -358,9 +361,25 @@ export function weeksAgo(dateFrom: Date, dateTo: Date) {
 }
 
 export function getLocationLabel(location: Location) {
-  return location.county
-    ? `${location.county}, ${location.state_code}`
-    : location.state;
+  const { county: countyName, state_code, state: stateName } = location;
+  const stateCode = state_code.toUpperCase();
+  return countyName
+    ? `${getAbbreviatedCounty(countyName)} ${stateCode}`
+    : stateName;
+}
+
+function truncateCountyName(countyName: string) {
+  return countyName.length < 8
```

Comment:
nit: I think this should probably be `<= 8` since if it's 8 characters long, it doesn't make sense to truncate the last character and replace it with …

We could almost maybe even do `<=9` since '…' is wider than a normal letter. 🤷